ui/src/components/welcome/QasWelcome.vue

```diff
@@ -1,13 +1,19 @@
 <template>
   <div class="q-mb-xl qas-welcome text-left">
-    <h3 class="text-grey-9 text-h3">
-      {{ welcomeMessage }}<span v-if="firstName">, {{ firstName }}</span>
-    </h3>
+    <div class="items-center justify-between row">
+      <div>
+        <h3 class="text-grey-9 text-h3">
+          {{ welcomeMessage }}<span v-if="firstName">, {{ firstName }}</span>
+        </h3>
 
-    <div class="text-caption text-grey-8">{{ currentDay }}</div>
+        <div class="text-caption text-grey-8">{{ currentDay }}</div>
+      </div>
+
+      <qas-actions-menu v-if="hasActionsMenuProps" v-bind="actionsMenuProps" />
```

Comment:
Achei interessante o uso do `QasActionsMenu` como padrão, mas to pensando se não seria interessante ter também um slot aqui, caso do lado de fora seja necessário renderizar algum outro componente. Se eu não me engano outros componentes do Asteroid dão essa possibilidade, ex: `QasPageHeader`, `QasHeaderActions`, etc...

Reply:
alias não daria pra usar o `header-actions` ?

Reply:
Sim, daria para usar, porém ele quebra a linha do botão que utilizo. Pode ser um problema do tamanho máximo do botão, mas ficou estranho.

Reply:
mas não é exatamente o msm comportamento do `QasHeaderActions` ? se está quebrando não seria um bug no componente? oq eu acho errado é ter um componente q faz exatamente o msm
![image](https://github.com/bildvitta/asteroid/assets/29342573/ed635d9c-3dc0-41f0-87cf-411f71db448d)
![image](https://github.com/bildvitta/asteroid/assets/29342573/af317544-f1d8-43fd-b752-440cd31d7bf8)